twistedchecker/test/test_runner.py

```diff
@@ -77,14 +82,13 @@ def test_findUselessCheckers(self):
         Test for method findUselessCheckers
         """
         runner = Runner()
-        registeredCheckers = sum(runner.linter._checkers.values(), [])
+        registeredCheckers = sum(list(runner.linter._checkers.values()), [])
```

Comment:
Do you really need to explicitly call list() in these changes,
or can you leave the calls to list() out?  converters like 2to3 tend to put them in, but most times you don't need them.